### ISSUE TITLE
Screwdriver is not working with new doors

### ISSUE THREAD

**Issue by Fixer-007** [minetest/minetest_game#904]
... both left and right mouse buttons won't work when using screwdriver on new doors.

**Comment by sofar** [comment]
https://github.com/minetest/minetest_game/blob/master/mods/doors/init.lua#L327

The screwdriver is disabled for doors on purpose. The placement is already smart enough to properly rotate the door on placement. Rotating the door with the screwdriver would require a lot of code for essentially very, very little actual use.

Just dig up the door and re-place it in the right angle.

**Comment by PilzAdam** [comment]
See #451.

This is a regression with the new doors; the old behavior should be restored (maybe label this issue as "bug"?).

**Comment by sofar** [comment]
@pilzadam I actually intend to address this issue. Keeping it open is appropriate.

**Comment by kilbith** [comment]
Actually I fixed that very easily :

``` Lua
-- Top of file
screwdriver = screwdriver or {}

-- Inside node definitions
on_rotate = screwdriver.rotate_simple
```

**Comment by sofar** [comment]
I intend to allow full rotation of doors, including sideways, upside-down, etc.

**Comment by paramat** [comment]
I'm happy with screwdrivers being disabled for doors.

**Comment by rubenwardy** [comment]
partially fixed by #1230

**Comment by paramat** [comment]
The lack of complex rotations is not an issue so closing.
